context.go

```diff
@@ -1097,7 +1097,7 @@ func getLocalStringListFromScope(scope *parser.Scope, v string) ([]string, scann
 				Pos: assignment.EqualsPos,
 			}
 		default:
-			panic(fmt.Errorf("unknown value type: %d", assignment.Value.Type))
+			panic(fmt.Errorf("unknown value type: %d", assignment.Value.Type()))
```

Comment:
Is the go 1.10 compiler better about catching these?

Reply:
I think this is the `go vet` support in `go test`